Makefile

```diff
@@ -355,9 +358,13 @@ vendor: glide.lock
         glide install -strip-vendor && \
         chown $(shell id -u):$(shell id -u) -R vendor'
 
-## Build the calicoctl
+## Build calicoctl
 binary: $(CALICOCTL_FILES) vendor
```

Comment:
I rather liked using `make binary` to make the binary for your local environment (in my case OS X) - looks like this breaks that now.